content/blog/2019/03/2019-03-04-gsoc2019-announcement.adoc

```diff
@@ -0,0 +1,87 @@
+---
+:layout: post
+:title: "Jenkins is accepted to Google Summer Of Code 2019!"
+:tags:
+- gsoc
+- gsoc2019
+- events
+- community
+:author: oleg_nenashev
+:sig: gsoc
+---
+
+image:/images/gsoc/jenkins-gsoc-logo_small.png[Jenkins GSoC, role=center, float=right]
+
+On behalf of the Jenkins GSoC org team,
+I am happy to announce that the Jenkins project **has been accepted** to
+link:https://summerofcode.withgoogle.com/[Google Summer of Code 2019].
+This year we invite students and mentors to join the Jenkins community and work together
+on enhancing the Jenkins ecosystem.
+
+Just to provide some numbers, this is the biggest GSoC ever, 206 organizations participate in GSoC this year.
+And it will be hopefully the biggest year for Jenkins as well.
+We have 25 link:/projects/gsoc/2019/project-ideas[project ideas]
+and more than 30 potential mentors (and counting!).
+It is already more than in 2016 and 2018 all together.
+There are many plugins, SIGs and sub-projects which have already joined GSoC this year.
+And we have already received messages and first contributions from dozens of students, yey!
+
+**What's next?**
+GSoC is officially announced, and please expect more more students to contact projects in our
+link:/projects/gsoc/#contacts[Gitter channels and mailing lists].
+This year there will be less traffic in the Developer mailing lists until projects are announced, because communications move to SIG and sub-project channels. Student project proposal deadline is April 9th, and we will be working hard in order to help students to find interesting projects, to explore the area, and to prepare their project proposals.
+We will send more information to mentors and students in separate emails.
+
+All information about the Jenkins GSoC is available on its link:/projects/gsoc/[sub-project page].
+
+== I am a student. How do I apply?
+
+See the link:/projects/gsoc/students[Information for students] page for full application guidelines.
+
+We encourage interested students is to reach out to the Jenkins community early and to start exploring project ideas.
+All project ideas have chats and mailing lists referenced on their pages.
+We will be also organizing office hours sessions for student,
+and you can use these meetings to meet org admins and mentors and to ask questions.
+Also, join our link:https://gitter.im/jenkinsci/gsoc-sig[Gitter channel] and the
+link:https://groups.google.com/forum/#!forum/jenkinsci-gsoc-all-public[mailing list]
+to receive information about such incoming events in the project.
+
+The application period starts on March 25th, but don't let it misguide you!
+Use time before application to discuss and improve your project proposals.
+We also recommend to study Jenkins and to do some contributions in the area of your project proposal.
```

Comment:
I would say "We also recommend you become familiar with Jenkins"

Reply:
agreed